news/doc.rst

```diff
@@ -1,6 +1,6 @@
 **Added:**
```

Comment:
I am not sure what happened here, but we likely need both news files, doc.rst and codecov.rst